notebooks/latent-consistency-models-image-generation/latent-consistency-models-image-generation.ipynb

```diff
@@ -708,14 +708,16 @@
    "source": [
```

Comment:
<p>not</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>